claim-check/code-samples/sample-2/client-consumer/EventReader.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Specialized;
+using Newtonsoft.Json.Linq;
+
+namespace ClientConsumer
+{
+    class EventReader : IReader
+    {
+        private string downloadDestination;
+       
+        private EventProcessorClient processor;
+
+        public void Configure()
+        {
+            Console.WriteLine("Validating settings...");
+            foreach (string option in new string[] { "EventHubConnectionString", "StorageConnectionString", "BlobContainerName", "DownloadDestination" })
+            {
+                if (string.IsNullOrEmpty(ConfigurationManager.AppSettings?[option]))
```

Comment:
This seems unnecessary.   You're walking configuration to throw out the value and then go back and read it 10 lines down.    Why not just read them in and then do a validation pass?   That would be the pattern that I would expect, as a developer looking at a sample.